Emma/emma_libs/memoryManager.py

```diff
@@ -326,8 +323,15 @@ def _createTeamScalePath(memEntryRow):
             createTeamScaleReports()
 
             if image:
-                startRegion = input("enter the beginning of memory region to plot (in hex) \n")
-                endRegion = input("enter the end of memory region to plot (in hex) \n")
-                createGraphics(int(startRegion, 16), int(endRegion, 16))
+                if noprompt:
+                    sc().wwarning("no prompt is active. No svg report will be created")
+                else:
+                    startRegion = input("Enter the start address of the region to be plotted (start with `0x` for hex; otherwise dec is assumed) \n")
+                    if "0x" in startRegion:
```

Comment:
Is not sufficient. use `.startswith()`

Reply:
Also check if all following characters are valid hex (also accept capital letters).